.gitignore

```diff
@@ -169,3 +169,7 @@ miniapps/nurbs/mesh.*
 miniapps/nurbs/sol.*
 miniapps/nurbs/mode_*
 miniapps/nurbs/Example1*
+
+# Unit test binary and outputs
+test/unittest/output_meshes
```

Comment:
I think we should go with plural: `tests` like `examples` or `miniapps`

Reply:
Done.

Reply:
This file (`.gitignore`) still needs to be updated: `test` --> `tests`.

Also, I prefer `tests/unit` (or `tests/unit_tests`) instead of `tests/unittest` - not sure what others think.